CoreScriptsRoot/Modules/VR/Panel3D.lua

```diff
@@ -781,7 +781,9 @@ local function onCameraChanged(prop)
 			currentHeadScale = workspace.CurrentCamera.HeadScale
 		end)
 		for i, v in pairs(panels) do
-			v:OnHeadScaleChanged(currentHeadScale)
```

Comment:
You should discard lines 784-786, I just fixed this in PR 411 (https://github.com/ROBLOX/Core-Scripts/pull/411)